unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/WearableGridController.cs

```diff
@@ -268,6 +286,65 @@ private async UniTask<int> RequestWearablesAndShowThem(int page, CancellationTok
             return 0;
         }
 
+        private async UniTask<IEnumerable<WearableItem>> FetchCustomWearableItems(CancellationToken cancellationToken)
```

Comment:
Right the same concerns as in the previous thread

Reply:
Done